x-pack/plugin/core/src/main/resources/ecs-core-mappings.json

```diff
@@ -0,0 +1,318 @@
+{
```

Comment:
It would be really nice to have this in yaml format, similar to what I did in https://github.com/elastic/elasticsearch/pull/89743/files This would allow us to add comments to each field on why it is there (or even why something is left out). Unfortunately what I had in https://github.com/elastic/elasticsearch/pull/89743 does not work as I think all the template tooling inside Elasticsearch is not made for yaml, even though the API can accept yaml.

Reply:
Considering this to be beyond the context of this PR 😕